ext/recaptcha/CRM/Utils/ReCAPTCHA.php

```diff
@@ -51,8 +52,7 @@ public static function &singleton() {
    * Check if reCaptcha settings is avilable to add on form.
    */
   public static function hasSettingsAvailable() {
-    $config = CRM_Core_Config::singleton();
-    if ($config->recaptchaPublicKey == NULL || $config->recaptchaPublicKey == "") {
+    if (empty(\Civi::settings()->get('recaptchaPublicKey'))) {
       return FALSE;
```

Comment:
Ditto - the whole function could be `return !(bool) \Civi::settings()->get('recaptchaPublicKey');`, assuming "0" isn't a valid recaptcha key.

Reply:
Agreed, thanks @demeritcowboy